kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/validation/impl/PaymentRequestTaxAreaValidation.java

```diff
@@ -0,0 +1,53 @@
+package edu.arizona.kfs.module.purap.document.validation.impl;

```

Comment:
I like how this class runs through multiple checks rather than returning after the first failure.